src/Run/Setup.cs

```diff
@@ -196,27 +223,46 @@ public int ExecuteCommand(string commandSelectedByUser, List<string> parametersS
             {
                 runQuiet = runQuietValue.Equals("true", StringComparison.OrdinalIgnoreCase);
             }
+            string whatIfValue;
+            bool whatIf = false;
+            if (ToolSettings.TryGetValue(WhatIfReservedKeyword, out whatIfValue))
+            {
+                if (string.IsNullOrEmpty(whatIfValue))
+                {
+                    ToolSettings.TryGetValue(DryRunReservedKeyword, out whatIfValue);
+                }
+                whatIf = whatIfValue.Equals("true", StringComparison.OrdinalIgnoreCase);
+            }
+
             CompleteCommand commandToRun = BuildCommand(commandSelectedByUser, parametersSelectedByUser);
             if (commandToRun != null)
             {
-                if (!runQuiet)
+                int result = 0;
+
+                if (whatIf)
                 {
-                    PrintColorMessage(ConsoleColor.DarkYellow, "Running: {0} {1}", commandToRun.ToolCommand, commandToRun.ParametersCommand);
+                    PrintColorMessage(ConsoleColor.Yellow, "Showing command, will not execute:");
+                    PrintColorMessage(ConsoleColor.Yellow, $"Command: {commandToRun.ToolCommand} {commandToRun.ParametersCommand}");
```

Comment:
In the spirit of whatif/dry-run I would must change the text to "Would execute: ". Also knowing out I would use this I would also suggest putting the command on a newline by-itself so it would be easier to copy.